apps/explorer/.env

```diff
@@ -24,6 +24,7 @@ NX_TENDERMINT_WEBSOCKET_URL = "wss://lb.testnet.vega.xyz/tm/websocket"
 NX_VEGA_URL = "https://lb.testnet.vega.xyz/query"
 NX_VEGA_ENV = 'TESTNET'
 NX_VEGA_REST = 'https://lb.testnet.vega.xyz/datanode/rest'
+NX_VEGA_NETWORKS = "TESTNET=https://explorer.fairground.wtf;MAINNET=https://explorer.vega.xyz"
```

Comment:
chore (blocking): if we need a string to be parsed into code let's use JSON rather than this.

Reply:
changed this to JSON, thanks!